source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -0,0 +1,343 @@
+.. _minio-mc-ilm-tier-add:
+
+===================
+``mc ilm tier add``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ilm tier add
+
+.. versionchanged:: RELEASE.2022-12-24T15-21-38Z
+
+   :mc-cmd:`mc ilm tier add` replaces ``mc admin tier add``.
+
+Description
+-----------
+
+.. start-mc-ilm-tier-add-desc
+
+The :mc:`mc ilm tier add` command creates a new remote storage tier to a supported S3-compatible service for MinIO :ref:`Lifecycle Management: Object Transition ("Tiering")<minio-lifecycle-management-expiration>`. 
+
+.. end-mc-ilm-tier-add-desc
+
+Supported S3 Services
+~~~~~~~~~~~~~~~~~~~~~
+
+:mc:`mc ilm tier add` supports *only* the following S3-compatible services as a remote target for object tiering:
+
+- MinIO
+- Amazon S3
+- Google Cloud Storage
+- Azure Blob Storage
+
+Permissions
+~~~~~~~~~~~
+
+MinIO requires the following administrative permissions on the cluster in which you create remote tiers for object transition lifecycle management rules:
+
+- :policy-action:`admin:SetTier`
+- :policy-action:`admin:ListTier`
+
+For example, the following policy provides permission for configuring object transition lifecycle management rules on any bucket in the cluster: 
+
+.. literalinclude:: /extra/examples/LifecycleManagementAdmin.json
+   :language: json
+   :class: copyable
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following example creates a new remote tier called ``WARM-MINIO-TIER`` on the ``myminio`` deployment.
+      The command creates a tier for a remote MinIO deployment located at the hostname ``https://warm-minio.com``.
+      
+      .. code-block:: shell
+         :class: copyable
+
+          mc ilm tier add minio myminio WARM-MINIO-TIER                     \
+                                        --endpoint https://warm-minio.com   \                       
+                                        --access-key ACCESSKEY              \
+                                        --secret-key SECRETKEY              \
+                                        --bucket mybucket                   \
+                                        --prefix myprefix/ 
+
+      Lifecycle management rules on the ``myminio`` deployment can use the new tier to transition objects into the remote location's ``myprefix/`` prefix in the ``mybucket`` bucket.
+
+   .. tab-item:: SYNTAX
+   
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ilm tier add TIER_TYPE                    \
+                         TARGET                       \
+                         TIER_NAME                    \
+                         --endpoint string            \
```

Comment:
Technically optional for non-S3/MinIO.

I wonder if *just* for this, we should have more tabs:

SYNTAX (S3)
SYNTAX (GCP)
SYNTAX (AZURE)

it means the syntax tab tree can get kinda long, which we don't have a design for though...

Or we could have three sub-blocks under syntax, like

```
# MinIO, AWS, or other S3-compatible remote
mc ilm tier add s3  TARGET TIER_NAME \
      --endpoint string \
     ...

# Google Cloud Storage Remote Target
mc ilm tier add gcp TARGET TIER_NAME \
      --params [--params]
```

etc.

breaks the copy-ability a bit, but maybe that's fine?

Reply:
We have examples for each in the Examples section down the page, so I'm inclined to not add them here.

Tried an alternative of adding a note and referencing the TIER_TYPE parameter section, where the parameters required are detailed by vendor.